examples/Leptonica/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+cmake_minimum_required(VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-Leptonica)
+
+# download Leptonica
+hunter_add_package(Leptonica)
+find_package(Leptonica CONFIG REQUIRED)
+add_executable(example example.c)
+
+include_directories(${Leptonica_INCLUDE_DIRS})
+target_link_libraries(example ${Leptonica_LIBRARIES})
```

Comment:
Don't need this. It should be just `target_link_libraries(example Leptonica::Leptonica)`.